UnityProject/Assets/Scripts/Objects/Pipes/Devices/Filter.cs

```diff
@@ -12,24 +14,7 @@ public class Filter : MonoPipe
 
 		private MixAndVolume IntermediateMixAndVolume = new MixAndVolume();
 
-		public Dictionary<string, Gas> CapableFiltering = new Dictionary<string, Gas>()
-		{
-			{"O2",Gas.Oxygen},
-			{"N2",Gas.Nitrogen},
-			{"PLS",Gas.Plasma},
-			{"CO2",Gas.CarbonDioxide},
-			{"N2O",Gas.NitrousOxide},
-			{"H2",Gas.Hydrogen},
-			{"H2O",Gas.WaterVapor},
-			{"BZ",Gas.BZ},
-			{"MIAS",Gas.Miasma},
-			{"NO2",Gas.Nitryl},
-			{"TRIT",Gas.Tritium},
-			{"HN",Gas.HyperNoblium},
-			{"STIM",Gas.Stimulum},
-			{"PLX",Gas.Pluoxium},
-			{"FRE",Gas.Freon},
-		};
+		public Dictionary<string, GasSO> CapableFiltering;
```

Comment:
Curious why we assign value in awake now? Also can probably be static, right?

Reply:
Had to move it to awake due to constructor violations with getting the GasSO's via the singleton.

Suppose it could be made static

Reply:
Yeah, should've read more changes first. Thanks. Yeah there's a few lists here that could be made static, if we care about saving an insignificant amount of memory.. Up to you.